docs/src/prototypes/dropdowns/dataMocks.ts

```diff
@@ -0,0 +1,53 @@
+export interface AtMention {
+  header: string
+  image: string
+  content: string
+}
+
+export const atMentionItems: AtMention[] = [
```

Comment:
```suggestion
export const atMentionItems: AtMentionItem[] = [
```

Type name is not represent the object, `AtMentionItem` should be more clear

Reply:
done, thanks